src/routes/RemediationDetails.js

```diff
@@ -12,6 +12,8 @@ import { addNotification } from '@redhat-cloud-services/frontend-components-noti
 import UpsellBanner from '../components/Alerts/UpsellBanner';
 import DeniedState from '../components/DeniedState';
 import classnames from 'classnames';
+import ActivityDetails from '../components/ActivityDetails';
```

Comment:
Can these top-level components be put into https://github.com/RedHatInsights/insights-remediations-frontend/tree/master/src/routes along with other top-level page components?